content/content-strategy/manage-content-requests/plan-content/who.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: text/textblock
+---
```

Comment:
missing param `section`  - needs to refer to the heading above it.

Reply:
Trisha: done